src/components/attendee/AttendeeHeader/AttendeeHeader.tsx

```diff
@@ -70,30 +73,51 @@ export const AttendeeHeader: React.FC<AttendeeHeaderProps> = ({
 
   const { userWithId } = useUser();
 
+  const isNarrow = isTablet || isMobile;
+
   const renderedCaptions = useMemo(
     () =>
-      Object.entries(tabCaptions).map(([key, label]) => (
-        <Button onClick={() => handleOverlayOpen(key)} key={key}>
-          {label}
+      !isNarrow ? (
+        Object.entries(tabCaptions).map(([key, label]) => (
+          <Button onClick={() => handleOverlayOpen(key)} key={key}>
+            {label}
+          </Button>
+        ))
+      ) : (
+        <Button variant="primaryAlternate" onClick={show}>
+          Menu
         </Button>
-      )),
-    [handleOverlayOpen]
+      ),
+    [handleOverlayOpen, show, isNarrow]
   );
 
   if (!userWithId) return null;
 
+  const containerClasses = classNames(CN.container, {
+    [CN.narrow]: isNarrow,
+  });
+
   return (
     <header className={headerClassnames}>
-      <div className={CN.container}>
-        <div>
-          {backButtonSpace ? (
-            <Button onClick={goBack}>
-              <FontAwesomeIcon icon={faArrowLeft} /> Leave
-            </Button>
-          ) : (
-            <Button>{space?.name ?? `This ${SPACE_TAXON.title}`}</Button>
-          )}
-        </div>
+      <div className={containerClasses}>
+        {backButtonSpace ? (
+          <Button
+            onClick={goBack}
+            variant={isNarrow ? "primaryAlternate" : "primary"}
+          >
+            <FontAwesomeIcon icon={faArrowLeft} /> Leave{STRING_SPACE}
+            {formatFullTimeLocalised(Date.now())}
```

Comment:
[NOTE] There is also `currentMilliseconds()` in `time.ts`. Don't know if it will be better or worse to directly use `Date` and `new Date`.

Reply:
used `currentMiliseconds` in this case [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)